<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Investigation Tracker</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Save and clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quitButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>30</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>RXR</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>60</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>NHS</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="rxrEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="contactBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>131</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Contact Type</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>113</width>
       <height>157</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="newContact">
        <property name="text">
         <string>Clinic - new</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="fuContact">
        <property name="text">
         <string>Clinic - follow up</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="critcareContact">
        <property name="text">
         <string>Critical care</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="adviceContact">
        <property name="text">
         <string>Advice</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="wardContact">
        <property name="text">
         <string>Ward</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="preclinicContact">
        <property name="text">
         <string>Pre clinic Ix</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radiologyContact">
        <property name="text">
         <string>Radiology meeting</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPlainTextEdit" name="notesEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="diagnosisBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>151</width>
      <height>451</height>
     </rect>
    </property>
    <property name="title">
     <string>Diagnosis</string>
    </property>
    <widget class="QPlainTextEdit" name="diagEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>390</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>141</width>
       <height>364</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="asthmaDiag">
        <property name="text">
         <string>Asthma</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="copdDiag">
        <property name="text">
         <string>COPD</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="bronchiectasisDiag">
        <property name="text">
         <string>Bronchiectasis</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="obDiag">
        <property name="text">
         <string>Obliterative bronchiolitis</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ildDiag">
        <property name="text">
         <string>ILD</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="osaDiag">
        <property name="text">
         <string>Sleep apnoea</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="coughDiag">
        <property name="text">
         <string>Cough</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="hypoventDiag">
        <property name="text">
         <string>Hypoventilation</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="sarcoidDiag">
        <property name="text">
         <string>Sarcoidosis</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="trachDiag">
        <property name="text">
         <string>Tracheomalacia</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="vteDiag">
        <property name="text">
         <string>Thromboembolic disease</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="critcareDiag">
        <property name="text">
         <string>Critical care follow up</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="lvfDiag">
        <property name="text">
         <string>Heart failure</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="funcDiag">
        <property name="text">
         <string>Functional</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="queryDiag">
        <property name="text">
         <string>Uncertain</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="otherDiag">
        <property name="text">
         <string>Other</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="investigationBox">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>151</width>
      <height>431</height>
     </rect>
    </property>
    <property name="title">
     <string>Investigation</string>
    </property>
    <widget class="QPlainTextEdit" name="ixEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>370</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>135</width>
       <height>341</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QCheckBox" name="pftIx">
        <property name="text">
         <string>PFTs</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="walkIx">
        <property name="text">
         <string>6 min walk</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="sleepIx">
        <property name="text">
         <string>Sleep study</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="oasysIx">
        <property name="text">
         <string>Oasys diary</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="pefrIx">
        <property name="text">
         <string>Other PEFR diary</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ctIx">
        <property name="text">
         <string>CT thorax</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="igeIx">
        <property name="text">
         <string>IgE</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="funcabsIx">
        <property name="text">
         <string>Functional Abs</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="asperIx">
        <property name="text">
         <string>Aspergillus IgG</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="serolIx">
        <property name="text">
         <string>Autoimmune serology</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="neuroIx">
        <property name="text">
         <string>Neurophysiology</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="bronchIx">
        <property name="text">
         <string>Bronchoscopy</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="echoIx">
        <property name="text">
         <string>Echocardiogram</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ettIx">
        <property name="text">
         <string>Exercise tolerance test</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="otherIx">
        <property name="text">
         <string>Other</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="disposalBox">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>10</y>
      <width>121</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Disposal</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>102</width>
       <height>157</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QRadioButton" name="dischDisp">
        <property name="text">
         <string>Discharge</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="fuDisp">
        <property name="text">
         <string>Follow up</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="letterDisp">
        <property name="text">
         <string>Results letter</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="dnadischDisp">
        <property name="text">
         <string>DNA - discharge</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="dnafuDisp">
        <property name="text">
         <string>DNA - reappoint</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="otherDisp">
        <property name="text">
         <string>Other / pending</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="admitDisp">
        <property name="text">
         <string>Admit from clinic</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="referralBox">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>151</width>
      <height>341</height>
     </rect>
    </property>
    <property name="title">
     <string>Referrals</string>
    </property>
    <widget class="QPlainTextEdit" name="refEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>128</width>
       <height>249</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QCheckBox" name="physioRef">
        <property name="text">
         <string>Physiotherapy</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="rehabRef">
        <property name="text">
         <string>Pulmonary rehab</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="nurseRef">
        <property name="text">
         <string>Respiratory nurse</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="saltRef">
        <property name="text">
         <string>SALT</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="heartRef">
        <property name="text">
         <string>Cardiology</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="entRef">
        <property name="text">
         <string>ENT</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="localRef">
        <property name="text">
         <string>Local chest physician</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="asthmaRef">
        <property name="text">
         <string>Tertiary airways clinic</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ventRef">
        <property name="text">
         <string>Ventilation clinic</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ildRef">
        <property name="text">
         <string>ILD clinic</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="otherRef">
        <property name="text">
         <string>Other</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QCheckBox" name="radmeetCheck">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>420</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>For radiology meeting</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Notes</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nhsEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>220</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Follow up time</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fuEdit">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>240</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMask">
     <string>09</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="juniorCheck">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>390</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Seen by junior</string>
    </property>
   </widget>
   <zorder>saveButton</zorder>
   <zorder>quitButton</zorder>
   <zorder>contactBox</zorder>
   <zorder>notesEdit</zorder>
   <zorder>diagnosisBox</zorder>
   <zorder>investigationBox</zorder>
   <zorder>disposalBox</zorder>
   <zorder>referralBox</zorder>
   <zorder>radmeetCheck</zorder>
   <zorder>label_3</zorder>
   <zorder>nhsEdit</zorder>
   <zorder>rxrEdit</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_4</zorder>
   <zorder>fuEdit</zorder>
   <zorder>juniorCheck</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>827</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuInvestigation_Tracker">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave_and_clear"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuInvestigation_Tracker"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSave_and_clear">
   <property name="text">
    <string>Save and clear</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
